awscli/examples/rum/update-app-monitor.rst

```diff
@@ -0,0 +1,10 @@
+**To update the configuration of an existing app monitor**
+
+The following ``update-app-monitor`` updates the configuration of an existing app monitor. If the command succeeds, no output is returned. ::
+
+	aws rum update-app-monitor \
```

Comment:
Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general